tests/spdx/writer/tagvalue/test_creation_info_writer.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: SPDX Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+import datetime
+from unittest.mock import mock_open, patch, call
+
+import pytest
+
+from spdx.model.document import CreationInfo
+from tests.spdx.fixtures import creation_info_fixture, actor_fixture
+
+from spdx.writer.tagvalue.creation_info_writer import write_creation_info
+
+
+@pytest.mark.parametrize("creation_info, expected_calls",
+                         [(creation_info_fixture(), [call("SPDXVersion: SPDX-2.3\n"), call("DataLicense: CC0-1.0\n"),
+                                                     call("SPDXID: SPDXRef-DOCUMENT\n"),
+                                                     call("DocumentName: documentName\n"),
+                                                     call("DocumentNamespace: https://some.namespace\n"),
+                                                     call("DocumentComment: documentComment\n"),
+                                                     call("\n## External Document References\n"), call(
+                                 "ExternalDocumentRef: DocumentRef-external https://namespace.com SHA1: 71c4025dd9897b364f3ebbb42c484ff43d00791c\n"),
+                                                     call("\n"), call("## Creation Information\n"),
+                                                     call("LicenseListVersion: 3.19\n"),
+                                                     call("Creator: Person: creatorName (some@mail.com)\n"),
+                                                     call("Created: 2022-12-01T00:00:00Z\n"),
+                                                     call("CreatorComment: creatorComment\n")]),
+                          (CreationInfo(spdx_version="SPDX-2.3", spdx_id="SPDXRef-DOCUMENT", creators=[actor_fixture()],
+                                        name="Test document", document_namespace="https://namespace.com",
+                                        created=datetime.datetime(2022, 3, 10)),
+                           [call("SPDXVersion: SPDX-2.3\n"), call("DataLicense: CC0-1.0\n"),
+                            call("SPDXID: SPDXRef-DOCUMENT\n"), call("DocumentName: Test document\n"),
+                            call("DocumentNamespace: https://namespace.com\n"), call("\n"),
+                            call("## Creation Information\n"), call("Creator: Person: actorName (some@mail.com)\n"),
+                            call("Created: 2022-03-10T00:00:00Z\n")])])
+def test_creation_info_writer(creation_info, expected_calls):
+    m = mock_open()
+    with patch("{}.open".format(__name__), m, create=True):
+        with open("foo", "w") as h:
+            write_creation_info(creation_info, h)
```

Comment:
I get that they are throw-away variables, but can we give `m` and `h` slightly more descriptive names?